aws_marketplace/creating_marketplace_products/models/build-model-package-for-listing-in-aws-marketplace.ipynb

```diff
@@ -0,0 +1,1122 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.                {"features": [5.1, 3.5, 1.4, 0.2]},
</pre><p>Nit - explain where this test data came from, and what it means. What is being predicted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>